### PR TITLE
Fixing Subtract Function

### LINKED ISSUE #5: subtract
## 5. Add `subtract`

__As a site visitor, I want to execute the function `subtract`__

> assigned to Abel

### Repo

- [x] developed on a branch named `subtract`
- [x] finish the function 'subtract.js`

### PR DESCRIPTION
**Pull request by abelRoland**
This closes #5

### CODE REVIEW

File: scripts/handlers/subtract.js [hunk 1]
```diff
@@ -83,10 +83,26 @@ function subtractHandler() {
...
 
+  const input = prompt('Write Some Number');
+  if (input === null || input === "") {
+    alert('Please enter a valid number');
+  } else {
+    let inputConfirmed = +input;
+    if (Object.is(inputConfirmed, NaN)) { alert('Be sure you are entering a number! Try again. '); }
+    else {
+      // subtract the last result by the user's number and reassign lastResult
+      const result = subtract(inputConfirmed, lastResult);
```
Comment (Sayed94h) [inline scripts/handlers/subtract.js]: Hi Abel, 
there is a shared variable called 'lastResult'. Here in this line, the variable 'lastResult' should be reassigned instead creating a new const variable 'result'. And also the lastResult should be subtracted from the user's input(just reverse the arguments inside the function).

Thank you

**Review by Sayed94h — CHANGES_REQUESTED** [review]
Hi Abel, there is a small issue in the code. I added comment for to make it clear.
Thank you. Greetings

File: scripts/handlers/subtract.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ function subtract(x, y) {
   if (typeof x !== 'number') { throw new TypeError('x'); }
   if (typeof y !== 'number') { throw new TypeError('y'); }
 
-  const result = _;
+  const result = x - y;
 
   if (typeof result !== 'number') { throw new TypeError('result'); }
   return result;
```
Comment (Sayed94h) [inline scripts/handlers/subtract.js:18]: Below this line there should be a "}" to close the subtract(x, y) function

Comment (abelRoland) [reply scripts/handlers/subtract.js:18]: ok. Thanks

**Comment by abelRoland** [comment]
I believe that now is ready to be merged.

**Pull request merged**